ledger/metrics/src/main/scala/com/daml/metrics/DatabaseMetrics.scala

```diff
@@ -5,23 +5,14 @@ package com.daml.metrics
 
 import com.codahale.metrics.{MetricRegistry, Timer}
 
-final class DatabaseMetrics private[metrics] (
+class DatabaseMetrics private[metrics] (
```

Comment:
Did you drop the `final` for some reason in particular?

Reply:
Yes, because I want to maintain the hierarchy for `store_ledger_entry` sub-metrics. For that I extended from `DatabaseMetrics` [here](https://github.com/digital-asset/daml/pull/6163/files#diff-c455b5ef86f3eac1e49aba3a837c4d95R337-R354).